tutorials/pages/create-nft-with-iota-sdk.md

```diff
@@ -0,0 +1,219 @@
+# Create an NFT with the IOTA SDK and IPFS in NodeJs
+
+In this tutorial, you will upload your desired image to [IPFS](https://ipfs.tech/) and create an NFT from the unique IPFS URL on the Shimmer testnet. This tutorial will build on the tutorial "Setup testnet address and receive funds from the faucet" ([IOTA Wiki PR under review](https://wiki.iota.org/tutorials/wallet-setup)). If you haven't completed it, please do so, and afterward, come back here, or make sure you have a prefunded wallet. You will use the same folder and development environment as in the mentioned prerequired tutorial, allowing you to reuse the Stronghold account with funds on it.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/).
```

Comment:
Could we mention the version used here? Makes it easier to check which tutorial need updates and so on. Same for the other tutorial

Reply:
Sorry I was unclear 😓 I meant the SDK version mostly. But having versions for the devs makes sense too 👍

Reply:
Ah.. got it.

Reply:
I see no changes

Reply:
added now.